holopy/core/io/vis.py

```diff
@@ -72,6 +72,48 @@ def show(o, scaling='auto', vert_axis='x', horiz_axis='y',
     else:
         raise VisualizationNotImplemented(o)
 
+def save(o, names, scaling='auto', vert_axis='x', horiz_axis='y',
+                    depth_axis='z', colour_axis='illumination'):
+    """
+    Saves a hologram or reconstruction to (a) file(s).
+
+    Parameters
+    ----------
+    o : xarray.DataArray or ndarray
+        Object to visualize.
+    names : str
+        Name(s) of the file(s). If there is only one image contained (e.g.
+        hologram), the name will be used as a file name. If o contains more
+        plottable images (e.g. reconstruction volume), this string will be
+        formatted to get a counting index inserted. Example:
+        "reconstruction-%i.png"
+    scaling : (float, float), optional
+        (min, max) value to display in image, default is full range of o.
+    vert_axis : str, optional
+        axis to display vertically, default x.
+    horiz_axis : str, optional
+        axis to display horizontally, default y.
+    depth_axis : str, optional
+        axis to scroll with arrow keys, default 'z'.
+    colour_axis : str, optional
+        axis to display as RGB channels in colour image, default illumination.
```

Comment:
If you can think of a pithy example, that would be great to add here!

Reply:
Are you thinking of an example for the parameters `depth_axis` and `colour_axis` or the method itself?